tokens/eth/0x960b236A07cf122663c4303350609A66A7B288C0.json

```diff
@@ -4,11 +4,11 @@
   "decimals": 18,
   "name": "Aragon",
   "ens_address": "",
-  "website": "https://aragon.org",
+  "website": "https://aragon.org/",
```

Comment:
what advantage do you see in adding this?

Reply:
It doesn’t change anything really, I like to keep the `/` explicit in URLs but it’s a personal preference. Happy to revert this change, let me know :-)

Reply:
yea - please remove it - my personal preference is to remove it as I see only disadvantages (consumes one more byte and IMHO looks ugly) and no advantages.

Reply:
Done!